src/main/java/seedu/address/logic/commands/TagFindCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.TagContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Keyword matching is case insensitive.
```

Comment:
Incorrect javadoc? I'm reviewing based on the assumption that this command will find an employee by the tag.

Reply:
Indeed, I forgot to change the javadoc after editing from the original "c-find" code. Thanks! I've edited this part.